### PR TITLE
docs: improve README documentation

### PR DESCRIPTION
**Pull request by jorgepiloto**
This pull-request improves the existing documentation of the `README.rst` file by adding a section named `Recommended workflows`. This sections has diagrams showing a different workflows for various common scenarios in GitHub.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -23,3 +23,46 @@ reused by projects in the Ansys ecosystem.
...
 For more information on available actions and how to use them, see
 `actions.docs.ansys.com <https://actions.docs.ansys.com>`_ .
+
+
+Recommended workflow strategy
+=============================
+
+The different actions provided by `ansys/actions
+<https://github.com/ansys/actions>`_ can be used to create a simple but
+effective workflows.
```
Comment (RobPasMue) [inline README.rst:33]:
```suggestion
effective workflow.
```

File: README.rst [hunk 2]
```diff
@@ -23,3 +23,46 @@ reused by projects in the Ansys ecosystem.
...
+|
+
+**Recommended workflow when performing a new release**
+This workflow outlines the recommended steps for performing a new software
+release, ensuring a smooth and well-documented process by deploying
+the stable documentation at the end of the workflow.
+
+.. image:: https://github.com/ansys/actions/blob/main/doc/source/_static/ci_cd_release.png
+
+
```
Comment (RobPasMue) [inline README.rst:68]: Why not link to the poster we have for more info?

**Review by RobPasMue — APPROVED** [review]
LGTM, some minor comments

**Review by MaxJPRey — APPROVED** [review]

File: README.rst (hunk 2 above)
Comment (jorgepiloto) [reply README.rst:68]: Sure! That's a great idea.

**Pull request merged**